src/controllers/legacycontrollermappingfilehandler.h

```diff
@@ -67,6 +72,13 @@ class LegacyControllerMappingFileHandler {
 
     bool writeDocument(const QDomDocument& root, const QString& fileName) const;
 
+#ifdef MIXXX_USE_QML
+    // Maximum target frame per request for a a screen controller
+    static constexpr int s_maxTargetFps = 240;
```

Comment:
am I right that the fps target is a per-controller setting? As in, will that be the user's choice or does the controller config author pick the value for the hardware?

Reply:
The controller config author picks this value and defines it in the manifest (e.g in the dummy demo, the "main" screen defines 20FPS, and the "artwork" screen defines 5). This max value is only here to prevent accidentally wrong value, although, I guess 240 is still quite high for onboard screens. Happy to adjust if you have another max value in mind